### DISCUSSION TITLE
SoftEther OpenVPN TLS-AUTH

### DISCUSSION THREAD

**Discussion by byeno16** [SoftEtherVPN/SoftEtherVPN#1798] [General]
Hello,
I have a softether developer edition server.
My friend at university cannot connect with tcp port 443 using openvpn protocol.

However, it can successfully connect to the openvpn server that I installed using the angristan openvpn-install script, again using the tcp 443 port.
The script I use: https://github.com/angristan/openvpn-install

When I use the script there is something like this in the client file
```
<tls-auth>
#
# 2048 bit OpenVPN static key
#
-----BEGIN OpenVPN Static key V1-----
blablabla
-----END OpenVPN Static key V1-----
</tls-auth>
```

But tls-auth is not available in SoftEther openvpn file.

Unable to connect without tls-auth I guess due to firewall.

How can I add tls-auth to Softether OpenVPN file?

Thanks for your help.

**Comment by shipitsin-ilia** [comment]
OpenVPN usually is listening 1194, not 443.

**Reply by byeno16** [reply]
No. You can set 443 tcp port. I can successfully connect via 443 tcp. 
Can't bypass firewall without tls auth.
